src/main/java/duke/command/FindCommand.java

```diff
@@ -0,0 +1,57 @@
+package duke.command;
+
+import duke.task.Task;
+import duke.task.TaskList;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class FindCommand extends Command {
+    private String taskName;
+
+    /**
+     * Constructor
+     * @param taskName - Name of task to search for
+     */
+    public FindCommand(String taskName) {
+        super(null);
```

Comment:
I noticed many of your command abstractions make calls to `super(null)`.

Maybe it would be easier to introduce an additional layer of abstraction, say, 'AddTaskCommand extends Command' and only commands that make use of the task instance variable would have it.